src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/DisposeMethodsShouldCallBaseClassDisposeTests.cs

```diff
@@ -775,10 +776,10 @@ End Sub
 End Class");
         }
 
-        [Fact, WorkItem(1654, "https://github.com/dotnet/roslyn-analyzers/issues/1654")]
-        public void BaseDisposeCalls_SomePaths_Diagnostic()
+        [Fact(Skip = "Analyzer isn't yet flow based."), WorkItem(1654, "https://github.com/dotnet/roslyn-analyzers/issues/1654")]
```

Comment:
❔ I'm guessing this never worked, but we just never knew it?

Reply:
I am not responsible for the change, it seems to be coming from some merge.